devel/py_example.md

```diff
@@ -0,0 +1,351 @@
+
+libnmstate.apply(
+{
+    Interface.KEY: [
+        {
+            Interface.NAME: 'eth1',
+            Interface.STATE: InterfaceState.UP
+            Interface.IPV4: {
+                InterfaceIPv4.ENABLED: True,
+                InterfaceIPv4.DHCP: True,
+                InterfaceIPv4.AUTO_ROUTES: False
+            },
+            Interface.IPV6: {
+                InterfaceIPV6.ENABLED: True,
+                InterfaceIPV6.DHCP: True,
+                InterfaceIPV6.AUTOCONF: True,
+                InterfaceIPv6.AUTO_ROUTES: False
+            },
+        }
+    ]
+})
+```
+
+## Ethernet
+
+### Change Ethernet MTU
+
+```python
+import libnmstate
+from libnmstate.schema import Interface
+
+libnmstate.apply(
+{
+    Interface.KEY: [
+        {
+            Interface.NAME: 'eth1',
+            Interface.STATE: InterfaceState.UP
+            Interface.MTU: 1460
+        }
+    ]
+})
+```
+
+## DNS
+
+```python
+import libnmstate
+from libnmstate.schema import Interface
+from libnmstate.schema import DNS
+
+libnmstate.apply(
+{
+    Interface.KEY: []
+    DNS.KEY: {
+        DNS.CONFIG: {
+            DNS.SEARCH: [
+                'example.org'
+            ],
+            DNS.SERVER: [
+                '192.0.2.251'
+            ]
+        }
+    }
+})
+```
+
+## Route configuration
+
+### Add static route entry
+
+```python
+import libnmstate
+from libnmstate.schema import Interface
+from libnmstate.schema import Route
+
+libnmstate.apply(
+{
+    Interface.KEY: [
+        {
+            Interface.NAME: 'eth1'
+        }
+    ]
+    Route.KEY: {
+        Route.CONFIG: [
+            {
+                Route.DESTINATION: "198.51.100.0/24",
+                Route.NEXT_HOP_INTERFACE: "eth1",
+                Route.NEXT_HOP_ADDRESS: "192.0.2.1",
+            }
+        ]
+    }
+})
+```
+
+### Change default gateway
+
+```python
+import libnmstate
+from libnmstate.schema import Interface
+from libnmstate.schema import Route
+
+libnmstate.apply(
+{
+    Interface.KEY: [
+        {
+            Interface.NAME: 'eth1'
+        }
+    ]
+    Route.KEY: {
+        Route.CONFIG: [
+            {
+                Route.STATE: Route.STATE_ABSENT,
+                Route.DESTINATION: "0.0.0.0/0",
+            }
+        ]
+        Route.CONFIG: [
+            {
+                Route.DESTINATION: "0.0.0.0/0",
+                Route.NEXT_HOP_INTERFACE: "eth1",
+                Route.NEXT_HOP_ADDRESS: "192.0.2.1",
+            }
+        ]
+    }
+})
+```
+
+### Remove all routes next hop to eth1
+
+```python
+import libnmstate
+from libnmstate.schema import Interface
+from libnmstate.schema import Route
+
+libnmstate.apply(
+{
+    Interface.KEY: [
+        {
+            Interface.NAME: 'eth1'
+        }
+    ]
+    Route.KEY: {
+        Route.CONFIG: [
+            {
+                Route.STATE: Route.STATE_ABSENT,
+                Route.NEXT_HOP_INTERFACE: "eth1",
+            }
+        ]
+    }
+})
+```
+
+## Bond
+
```

Comment:
An explicit `TBD` would make it clearer that there is stuff here to be documented.